test/utils.js

```diff
@@ -7,7 +7,7 @@ export const checkFields = (t, object, fields) => {
 }
 
 const generatePort = (() => {
-  let portNum = 9000;
+  let portNum = parseInt(9000 + (65535 - 9000) * Math.random(), 10);
```

Comment:
Do you mind removing this? It seems out of scope of this PR

Reply:
My bad, on my PC ports 9000-9001 are busy by [Portainer](https://www.portainer.io/), so I couldn't run tests (before commits)
I rolled it back